omc3/optics_measurements/data_models.py

```diff
@@ -0,0 +1,185 @@
+"""
+Data Models
+-----------
+
+Models used in optics measurements to store and pass around data.
+"""
+from typing import Dict
+
+import numpy as np
+import pandas as pd
+
+import tfs
+from omc3.definitions.constants import PLANES
+from omc3.optics_measurements import dpp, iforest
+from omc3.optics_measurements.constants import BPM_RESOLUTION, AMPLITUDE, CALIBRATION, ERR, ERR_CALIBRATION
+from omc3.utils import logging_tools
+
+LOGGER = logging_tools.get_logger(__name__)
+
+class InputFiles(dict):
+    """
+    Stores the input files, provides methods to gather quantity specific data
+
+    Public methods:
+        - ``get_dpps`` (plane)
+        - ``get_joined_frame`` (plane, columns, zero_dpp=False, how='inner')
+        - ``get_columns`` (frame, column)
+        - ``get_data`` (frame, column)
+    """
```

Comment:
I have a feeling maybe we could be a bit more descriptive in the docstring here?

Reply:
I am already happy to have moved this moster somewhere. More cleanup later